tests/drivers/dma/chan_blen_transfer/boards/mimxrt1024_evk.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_CODE_DATA_RELOCATION=y
+CONFIG_NOCACHE_MEMORY=y
```

Comment:
is `CONFIG_NOCACHE_MEMORY` still needed?

Reply:
yes, it can be removed